### PR TITLE
PLAT-78555: Handle resize event in Scrollable

### PR DESCRIPTION
**Pull request by ybsung**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)

VirtualList is not responsive when resizing a window.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

VirtualList is recalculated and rendered again when receiving `resize` event.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

This PR also covered the case when rotating a window.

### Links
[//]: # (Related issues, references)

PLAT-78555

### CODE REVIEW

File: packages/ui/Scrollable/Scrollable.js [hunk 1]
```diff
@@ -1243,6 +1250,12 @@ class ScrollableBase extends Component {
 		if (this.props.addEventListeners) {
 			this.props.addEventListeners(childRefCurrent.containerRef);
 		}
+
+		if (this.childRefCurrent.cc) { // For VirtualList
```
Comment (0x64) [inline packages/ui/Scrollable/Scrollable.js]: Is there any better way to distinguish the type of child component rather than checking `cc`?

Comment (ybsung) [reply packages/ui/Scrollable/Scrollable.js]: While we discussed this issue, we found that Scroller also need to be resizable to show or to hide scrollbars. I'll remove the condition.

**Comment by 0x64** [comment]
LGTM

**Comment by seunghoh** [comment]
GridListNative position is wrong when you focus on scroller button and rotate the view.

**Comment by 0x64** [comment]
@seunghoh Good point! A browser keeps `scrollTop` value for natively scrollable elements. But our components reset on resize (int this PR), so temporarily the scroll position may not match with components' internal status.

**Comment by seunghoh** [comment]
@ybsung on Mobile (Android and iPhone both) selected item is changed.
LIke, spot on item 63 on portrait, rotate device to landscape then selected item changed to item 3

**Comment by ybsung** [comment]
Hi @seunghoh 
When rotating a device, Spotlight is hidden now with my patch.

**Pull request merged**